coralproject/talk
# [next] Email/Password Improvements

## What does this PR do?

- All secret fields are now "hidden" by default
- All email fields now have autocomplete/spellcheck disabled with
  correct typings

## How do I test this PR?

Try to login on iOS, notice that the email field does not autocaptilize the first character.
